packages/schema/__tests__/util/ascendToPackageDir.ts

```diff
@@ -0,0 +1,17 @@
+import fs from "fs";
```

Comment:
Let's use snake_case for filenames

In general, let's follow https://google.github.io/styleguide/tsguide.html

Reply:
Oh, yeah sorry, let's do kebab-case for filenames.